Classes/Core/Functional/FunctionalTestCase.php

```diff
@@ -367,7 +367,49 @@ protected function setUp(): void
                     'Resources/Core/Functional/Extensions/private_container',
                 ]
             );
+
+            // Cache the Dependency Injection Container, which is dependent on the PackageStates file
+            $pstate_file = $this->instancePath. '/typo3conf/PackageStates.php';
```

Comment:
ohh yes, python it was :)

Reply:
There are some more minor cgl issues here. We do not enforce them yet in TF through tools, like for example for TYPO3 core. And I would prefere to have some method(s) instead of this kind of block here.

However, let it for now - I will have a detailed look on it after work with proper evaluation and testings. And eventually make the adjustments.

Reply:
will check later  and add the changes :) ..